helm_deploy/values-prod.yaml

```diff
@@ -10,7 +10,8 @@ generic-service:
 
   postgresDatabaseRestore:
     enabled: true
-    timeout: 7200
+    timeout: 3590
+    schedule: "5 0-5,20-23 * * 0-5"
```

Comment:
overnight only. The thinking here is that the job may fail if it runs at a time when there is a danger of e.g. activities or csips being created while a) the unique indexes have been dropped and b) the id sequences are set lower than items already in the mapping table  (this happens when nomis was copied to preprod significantly earlier than the postgres db) thus creating duplicates.

Reply:
This means the job runs asap (including on Sunday if Nomis becomes available soon enough). Looking at 1st dec, the copy of nomis was frozen at about 00:05 on Monday.